docker-compose.yaml

```diff
@@ -11,4 +11,4 @@ services:
     volumes:
       - .:/app
     ports:
-      - 3000:3000
+      - 3000:3002
```

Comment:
hint:
- host-port:docker-port

so it should be 3002:3000 not 3000:3002

the same in the previous file with port 80

Reply:
Sorry man I actually got it right first but changed it wrong. I changed the wrong order now.